src/tokens.ts

```diff
@@ -275,6 +276,7 @@ export const TOKEN_SYMBOLS_MAP = {
     symbol: "USDT",
     decimals: 6,
     addresses: {
+      [CHAIN_IDs.ALEPH_ZERO]: "0x4Ca4B85Ead5EA49892d3a81DbfAE2f5c2F75d53D",
```

Comment:
This one is USDC: https://evm-explorer.alephzero.org/address/0x4Ca4B85Ead5EA49892d3a81DbfAE2f5c2F75d53D

Reply:
Done - this was a copy paste issue during a pair programming session - the corrected version was on local 😓. Thanks for the callout!